packages/ripple-address-codec/package.json

```diff
@@ -21,12 +21,13 @@
   "prepublishOnly": "tslint -b ./ && jest",
   "scripts": {
     "build": "tsc -b",
-    "test": "jest",
+    "test": "jest --verbose false --silent=false --runInBand --detectOpenHandles ./src/*.test.js",
```

Comment:
Apparently `--detectOpenHandles` has "significant performance penalties and should only be used for debugging" - should this be removed from the default test command for now? 

(Maybe we could add a `test:debug` command which is has all the handy functions for figuring out what the heck the tests are doing?)

Reply:
Good catch

Reply:
(Whatever's decided here will apply to the other package.json's as well)